compute/instances/concepts.mdx

```diff
@@ -100,7 +100,7 @@ Creating an image allows you to make a complete backup of your Instance. One ima
 
 ## Instance
 
-An Instance is a computing unit, either virtual or physical, that provides resources to run your applications on. Currently, Scaleway offers the following Instance types: [General Purpose](#general-purpose-instance), [Development](#development-instance), [GPU](#gpu-instance), [Stardust](#stardust-instance) and [Enterprise](#enterprise-instance). For more information about the different ranges of Instances, see our [dedicated documentation](/compute/instances/reference-content/choosing-instance-type/).
+An Instance is a virtual computing unit that offers resources for running applications. It functions as a self-contained entity with its own operating system, RAM, and storage. You have the flexibility to tailor the Instance configuration according to your requirements, including the number of CPU cores, RAM capacity, and storage space. For further details on the various Scaleway Instance options, refer to our [Instance types documentation](/compute/instances/reference-content/choosing-instance-type/).
```

Comment:
```suggestion
An Instance is a virtual computing unit that offers resources for running applications. It functions as a self-contained entity with its own operating system, RAM, and storage. You have the flexibility to tailor the Instance configuration according to your requirements, including the number of vCPUs, RAM capacity, and storage. Refer to our [Instance types documentation](/compute/instances/reference-content/choosing-instance-type/) for further details on the various Scaleway Instance options.
```

- We offer "vCPUs" on the console rather than CPU cores. 
- A bit preferential, but "storage space" feels redundant and "storage" alone is more appropriate in this context I think (storage space sounds like Google Drive to me :) )
- Flip around last sentence for active voice + call to action